bin/setup

```diff
@@ -10,3 +10,9 @@ if [ -z "$CI" ]; then
 fi
 
 bundle exec rake dummy:db:create
+
+rails -v | grep -q '5.0'
+greprc=$?
+if [[ $greprc -eq 0 ]]; then
```

Comment:
@teoljungberg Yeah - if you haven't noticed, shell scripting isn't my strong point... Thanks for the tip!